hack/istio/install-topology-generator-demo.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+##############################################################################
+# install-scale-mesh-demo.sh
+#
+# Installs the kiali topology-generator demo application
+# https://github.com/kiali/demos/tree/master/topology-generator
+# Works on both openshift and non-openshift environments.
+##############################################################################
+
+: ${CLIENT_EXE:=oc}
+: ${DELETE_DEMOS:=false}
+: ${DELETE_CONFIG:=false}
+: ${TOPO:=topology-generator}
+: ${BASE_URL:=https://raw.githubusercontent.com/kiali/demos/master}
+: ${NUM_NS:=1}
+
+apply_network_attachment() {
+  NAME=$1
+cat <<NAD | $CLIENT_EXE -n ${NAME} apply -f -
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: istio-cni
+  labels:
+   generated-by: mimik
+NAD
+    cat <<SCC | $CLIENT_EXE apply -f -
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: ${NAME}-scc
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+supplementalGroups:
+  type: RunAsAny
+users:
+- "system:serviceaccount:${NAME}:default"
+- "system:serviceaccount:${NAME}:${NAME}"
+SCC
+}
+
+install_topology_generator_demo() {
+
+    if [ "${IS_OPENSHIFT}" == "true" ]; then
+        ${CLIENT_EXE} new-project ${TOPO}
+      else
+        ${CLIENT_EXE} create ns ${TOPO}
+    fi
+    ${CLIENT_EXE} label namespace  ${TOPO} istio-injection=enabled --overwrite=true
+
+    if [ "${IS_OPENSHIFT}" == "true" ]; then
+      apply_network_attachment ${TOPO}
+    fi
+
+    ${CLIENT_EXE} apply -n ${TOPO}  -f ${BASE_URL}/topology-generator/deploy/generator.yaml
+}
+
+generate_config() {
+
+  if [ "${IS_OPENSHIFT}" == "true" ]; then
+    for (( i=1; i<=$NUM_NS; i++ ))
+        do
+
+            apply_network_attachment n${i}
+    done
+  fi
+}
+
+while [ $# -gt 0 ]; do
+  key="$1"
+  case $key in
+    -c|--client)
+      CLIENT_EXE="$2"
+      shift;shift
+      ;;
+    -n|--num_ns)
+      NUM_NS="$2"
+      shift;shift
+      ;;
+    -g|--generate_config)
```

Comment:
here's another one with `_` - just make it `-`:

```suggestion
    -g|--generate-config)
```